### ISSUE TITLE
Orbit Key Controls

### ISSUE THREAD

**Issue by garethjohnsdesign** [zurb/foundation-sites#11408]
I'm trying using orbit on a site I'm working on

https://www.martinzahringer.com/photographs/self-service-oxfam/
https://github.com/garethjohnsdesign/martinzahringer.com

Clicking the slide advances the orbit as expected, hover the left and right keys don't work, I've read through the documentation but is slightly confusing of how to change the bindings

Can anyone give me a suggestion on how I can work this.

**Comment by DanielRuf** [comment]
I guess you mean https://github.com/zurb/foundation-sites/blob/develop/js/foundation.orbit.js#L40-L49

**Comment by garethjohnsdesign** [comment]
Yes but how do I make it so that the orbit slider is automatically selected?

**Comment by Owlbertz** [comment]
@garethjohnsdesign I am not sure what you want to achieve or what the issue is, but here is what I can tell you about thath.

At the end of the page you referenced you have the following code:
```html

<script>
document.body.onkeyup = function(e){
if (e.keyCode == '37') { window.location = ''; }
if (e.keyCode == '39') { window.location = ''; }
};
</script>
```
This causes the page to reload when you click either left or right arrow. So from what I can tell the issue is not caused by Orbit. When looking closely you can actually see that when pressing left/right arrow the previous/next image is being shown just before the page renders. If you remove that the behavior should work as expected: when the Orbit has focus, pressing either key will change the slides.

If you want to get rid of the following custom bindings and have the slider listen to page-wide keyboard events, you could use the following code:
```javascript
// Remove Foundation event listener
$('.orbit-container').off('keydown.zf.orbit');
// Attach page wide listener
$('body').on('keydown', function(e) {
    if (e.keyCode == '37') { $('[data-orbit]').foundation('changeSlide', false); }
    if (e.keyCode == '39') { $('[data-orbit]').foundation('changeSlide', true);}
});
```
Hope that helps.
